sdk/dapp-kit/src/index.ts

```diff
@@ -3,8 +3,10 @@
 
 export * from './hooks/useSuiClient.js';
 export * from './components/SuiClientProvider.js';
+export * from './components/WalletProvider.js';
```

Comment:
We'll likely just expose a `DAppKitProvider` hook that wraps `SuiClientProvider` and `WalletProvider`, but we'll do that in a different PR